src/views/product-integrations-landing/contexts/integrations-search-context.tsx

```diff
@@ -286,11 +286,13 @@ export const IntegrationsSearchProvider = ({
 				}
 			})
 
+			// If no types are selected, do not filter by type
 			let typeMatch = !typesCheckedArray.includes(true)
 			typesCheckedArray.forEach((checked, index) => {
+				// set typeMatch to true if the integrations type is selected
 				if (checked) {
 					const checkedType = types[index]
-					if (integration.integration_type === checkedType) {
+					if (integration.integration_type.slug === checkedType.slug) {
```

Comment:
This check was not working properly, and was preventing the filtering from displaying more than one element.  I am not entirely sure why this is the case, but I've reduced this down to compare the slugs, which produces the correct result.